modules/authentication-remove-kubeadmin.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-authentication.adoc
+
+[id="removing-kubeadmin-{context}"]
+= Removing the kubeadmin user
+
+Once an identity provider has been defined it is recommended to remove
+the `kubeadmin` user for security reasons.
+
+WARNING
+====
+If this procedure is followed before another user is a `cluster-admin`,
```

Comment:
s/If this procedure is followed/If you follow this procedure

Reply:
Corrected.